lib/collections/accounts.js

```diff
@@ -0,0 +1,113 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
```

Comment:
Is there a `SimpleSchema` in `reaction/lib/collections/schemas` or what am I importing?